master/internal/experiment.go

```diff
@@ -468,7 +487,10 @@ func (e *experiment) processOperations(
 	}
 	if err != nil {
 		ctx.Log().Error(err)
-		e.updateState(ctx, model.StoppingErrorState)
+		e.updateState(ctx, model.StateWithReason{
+			State:               model.StoppingErrorState,
+			InformationalReason: "encountering error",
```

Comment:
maybe 'encountered error' for tense consistency (maybe `fmt.Sprintf` the error in just so if someone does report it we don't need to immediately ask for the log to know what it was)